cirq-core/cirq/ops/clifford_gate_test.py

```diff
@@ -58,9 +58,12 @@ def _all_rotation_pairs():
         yield (px, flip_x), (pz, flip_z)
 
 
+@functools.cache
 def _all_clifford_gates():
```

Comment:
Line not touched but might as well add a return type annotation while we're at it.

Reply:
done

Reply:
and it made me realize this likely doesn't work since List is not hashable....so changed to tuple!